pliers/utils/scikit.py

```diff
@@ -42,10 +42,12 @@ def transform(self, stimulus_files):
             result = self.transformer.transform(stimulus_files).to_df()
         else:
             result = self.transformer.transform(stimulus_files, merge=False)
-            result = merge_results(result, flatten_columns=True)
+            result = merge_results(result, format='wide',
+                                   extractor_names=False)
```

Comment:
Could probably just put `metadata=False` and `timing=False` instead of the extra code before

Reply:
Point taken, but I think it's helpful to store those extra columns in `metadata_`, so I'll leave it as-is.

Reply:
Oh yea forgot about the `metadata_` attribute 👍